src/containers/Header/PreviewFlag.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function PreviewFlag({ name }) {
+    if(!name) {
+        return null
+    }
+
+    return (
+        <>
+            <span className="preview-flag">
+                { name }
+            </span>
+        </>
+  )
+}
```

Comment:
@queuing4oranges Add an empty new line at the end of the file. Its a unwritten rule :) (also github is pointing out that there is empty newline missing)